### PR TITLE
Mezon fe sort channel ctrl k

### PR DESCRIPTION
**Pull request by Tran-Binh-Duong**

### CODE REVIEW

File: libs/components/src/lib/components/SettingProfile/SettingRightClanProfile/settingUserClanProfileEdit.tsx [hunk 1]
```diff
@@ -67,15 +69,35 @@ const SettingRightClanEdit = ({
...
+			return;
 		}
+		handleUploadFile(client, session, '0', '0', imageAvatarResize?.name, imageAvatarResize).then((attachment: any) => {
+			setUrlImage(attachment.url ?? '');
+			if (attachment.url !== userProfile?.user?.avatar_url) {
+				setFlagOptionsTrue?.();
+			} else {
+				setFlagOptionsfalse?.();
+			}
+		});
```
Comment (sontung-nccvn) [inline libs/components/src/lib/components/SettingProfile/SettingRightClanProfile/settingUserClanProfileEdit.tsx]: use setFlagOption with param instead

File: libs/components/src/lib/components/ModalUserProfile/StatusProfile/index.tsx [hunk 2]
```diff
@@ -1,4 +1,5 @@
 import { Icons } from '@mezon/components';
+import { useMemberCustomStatus } from '@mezon/core';
 import { ChannelMembersEntity, useAppDispatch, userClanProfileActions } from '@mezon/store';
 import { Dropdown } from 'flowbite-react';
 import { OnlineStatus } from '../../../../../../ui/src/lib/Icons';
```
Comment (sontung-nccvn) [inline libs/components/src/lib/components/ModalUserProfile/StatusProfile/index.tsx]: use absolute imports

File: libs/components/src/lib/components/SettingProfile/SettingRightClanProfile/settingUserClanProfileEdit.tsx (hunk 1 above)
Comment (Tran-Binh-Duong) [reply libs/components/src/lib/components/SettingProfile/SettingRightClanProfile/settingUserClanProfileEdit.tsx]: Done

File: libs/components/src/lib/components/ModalUserProfile/StatusProfile/index.tsx (hunk 2 above)
Comment (Tran-Binh-Duong) [reply libs/components/src/lib/components/ModalUserProfile/StatusProfile/index.tsx]: Done

**Pull request merged**